core/source/org/libreoffice/ide/eclipse/core/actions/UnoTypePulldownAction.java

```diff
@@ -106,6 +111,14 @@ public boolean isValidSelection(IStructuredSelection pSelection) {
                 }
             }
         }
+        else {
+            Shell shell = Display.getDefault().getActiveShell();
+            MessageDialog.openError(shell, "Error", "Error occured\nYou need to create a LibreOffice project first to use the selection");
```

Comment:
You can directly call getShell() here, no need to create a variable for that.
Also please remove the "Error occured\n" from the error message. The dialog already has the type and title "Error".